frontend/index.html

```diff
@@ -3,9 +3,12 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>transcribee 🐝</title>
+    <title>transcribee</title>
     <link rel="stylesheet" href="/src/index.css" />
     <link rel="stylesheet" href="/node_modules/@fontsource/inter/latin.css" />
+
+    <link rel="icon" href="/favicon.ico" sizes="any">
+    <link rel="icon" href="/favicon.svg" type="image/svg+xml">
```

Comment:
```suggestion
    <link rel="icon" href="/favicon.ico" sizes="48x48">
    <link rel="icon" href="/favicon.svg" sizes="any" type="image/svg+xml">
```

See start of https://dev.to/masakudamatsu/favicon-nightmare-how-to-maintain-sanity-3al7 and the two linked comments:

> The reason is that the sizes="any" attribute no longer works for the ICO favicon file with Chromium browsers (as reported by [Doug Waldron back in March](https://dev.to/guitarzan/comment/25k7n) and [Darryl Noakes back in May](https://dev.to/darrylnoakes/comment/2724c)).